maintenance/health/servicehealthcheck/healthchecker.go

```diff
@@ -41,23 +43,19 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	name := splitRoute[1]
-	hcInterface, isIn := checks.Load(name)
-	if !isIn {
-		h.writeError(w, errors.New("health check not registered"), http.StatusNotFound, name)
+	if name == "all" {
```

Comment:
Please also add a check to the `RegisterHealthCheck` that prevents registering a check with the name `"all"`.